ios/Classes/SwiftPolarPlugin.swift

```diff
@@ -123,11 +111,20 @@ public class SwiftPolarPlugin:
     }
   }
 
+  private func initSdk() {
+    guard api == nil else { return }
+    api = PolarBleApiDefaultImpl.polarImplementation(
+      DispatchQueue.main, features: Set(PolarBleSdkFeature.allCases))
+
+    api.observer = self
+    api.powerStateObserver = self
+    api.deviceFeaturesObserver = self
+    api.deviceInfoObserver = self
+  }
+
   var searchSubscription: Disposable?
   lazy var searchHandler = StreamHandler(
     onListen: { _, events in
-      self.initApi()
```

Comment:
This should not be removed. It is possible to listen to this stream without calling a method which would result in the SDK being uninitialized.